pkg/controller/common/helpers.go

```diff
@@ -131,6 +133,39 @@ func MergeMachineConfigs(configs []*mcfgv1.MachineConfig, osImageURL string) (*m
 	}, nil
 }
 
+// PointerConfig generates the stub ignition for the machine to boot properly
+// NOTE: If you change this, you also need to change the pointer configuration in openshift/installer, see
+// https://github.com/openshift/installer/blob/master/pkg/asset/ignition/machine/node.go#L20
+func PointerConfig(ignitionHost string, rootCA []byte) (ign2types.Config, error) {
```

Comment:
Surely we should have this start out as spec3 now?

Reply:
A yep, I did a cherry-pick of the original patch instead of a revert of the revert, so I missed 22b8b24a39bfdcfab82e47ba9ebb3dc22e0e716d (and f6b693805f4fa941a74096ccad7e393878014105) on the first pass. Will add those in.